quinn-proto/src/assembler.rs

```diff
@@ -113,6 +113,7 @@ impl Assembler {
     }
 
     pub(crate) fn pop(&mut self) -> Option<(u64, Bytes)> {
```

Comment:
Should `read` call this rather than more or less duplicating its logic?

Reply:
Good call, fixed.